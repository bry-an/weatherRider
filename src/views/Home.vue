<template>
  <div>
    <auto-complete/>
    <google-map/>
  </div>
</template>

<script>
import AutoComplete from "@/components/AutoComplete";
import GoogleMap from "@/components/GoogleMap";

export default {
  components: {
    AutoComplete,
    GoogleMap
  }
};
</script>
