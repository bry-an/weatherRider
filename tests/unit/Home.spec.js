import { shallowMount } from "@vue/test-utils";
import Home from "@/views/Home.vue";

describe("home view", () => {
  const build = () => {
    const wrapper = shallowMount(Home);

    return {
      wrapper
    };
  };

  it("renders correctly", () => {
    const { wrapper } = build();

    expect(wrapper.html()).toMatchSnapshot();
  });
});
