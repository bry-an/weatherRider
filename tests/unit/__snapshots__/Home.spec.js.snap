// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home view renders correctly 1`] = `
<div>
  <auto-complete-stub></auto-complete-stub>
  <google-map-stub></google-map-stub>
</div>
`;
